src/cmake/Dashboard.cmake.in

```diff
@@ -0,0 +1,86 @@
+# Copyright (c) 2017-2022, Lawrence Livermore National Security, LLC and
+# other Axom Project Developers. See the top-level LICENSE file for details.
+#
+# SPDX-License-Identifier: (BSD-3-Clause)
+
+#------------------------------------------------------------------------------
+# Sets up variables for building and testing a configuration
+# and reporting the results to CDash
+#------------------------------------------------------------------------------
+
+set(CTEST_PROJECT_NAME "Axom")
+
+set(CTEST_SITE "LC gitlab")
+site_name(CTEST_SITE)
+
+# drop site options
+set(CTEST_SUBMIT_URL "https://$ENV{LC_CDASH_DROP_SITE}/submit.php?project=${CTEST_PROJECT_NAME}")
+
+# this is our token for cdash
+set(_auth_token "$ENV{LC_CDASH_TOKEN}")
+
+set(CTEST_CUSTOM_MAXIMUM_NUMBER_OF_ERRORS           "200" )
+set(CTEST_CUSTOM_MAXIMUM_NUMBER_OF_WARNINGS         "500" )
+set(CTEST_CUSTOM_MAXIMUM_PASSED_TEST_OUTPUT_SIZE    "104857600") # 100 MB
+set(CTEST_CUSTOM_COVERAGE_EXCLUDE                   "")
+set(CTEST_NIGHTLY_START_TIME                        "01:00:00 UTC")
+set(CTEST_USE_LAUNCHERS                             1)
+
+# list of additional warning messages to filter
+list(APPEND CTEST_CUSTOM_WARNING_EXCEPTION
```

Comment:
thousand thumbs up. thanks for figuring this out.

Reply:
Wow. 10 x 💯 👍